web/wp-content/themes/dctx/header.php

```diff
@@ -11,6 +11,16 @@
 	<link rel="icon" type="image/png" href="<?php echo ROOT; ?>favicon.png">
 
 	<?php wp_head(); ?>
+
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162125973-1"></script>
```

Comment:
:no_entry_sign: **Error**: Scripts must be registered/enqueued via wp_enqueue_script (*WordPress.WP.EnqueuedResources.NonEnqueuedScript*).